src/pages/blog/index.astro

```
@@ -7,15 +7,41 @@ import PostCard from "@components/PostCard.astro";
 import { SITE } from "@config";
 import externalArticles from "@data/external-articles.json";
 import Layout from "@layouts/Layout.astro";
+import { buildBreadcrumbList } from "@utils/jsonLd";
 import getSortedPosts from "@utils/getSortedPosts";
```

ESLint

[error] `@utils/getSortedPosts` import should occur before import of `@utils/jsonLd`

(import-x/order)